### PR TITLE
ADD Sapian as User of Cilium

### PR DESCRIPTION
**Pull request by arpagon**
Please ensure your pull request adheres to the following guidelines:

- [x] For first time contributors, read [Submitting a pull request](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#submitting-a-pull-request)
- [x] All code is covered by unit and/or runtime tests where feasible.
- [x] All commits contain a well written commit description including a title,
      description and a `Fixes: #XXX` line if the commit addresses a particular
      GitHub issue.
- [x] All commits are signed off. See the section [Developer’s Certificate of Origin](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#dev-coo)
- [x] Provide a title or release-note blurb suitable for the release notes.
- [x] Thanks for contributing!

Fixes: #issue-number

```release-note
<!-- Enter the release note text here if needed or remove this section! -->
```

### CODE REVIEW

**Comment by arpagon** [comment]
@xmulligan I already ammed  the Signed-off-by. thanks

**Review by xmulligan — APPROVED** [review]

**Comment by xmulligan** [comment]
Perfect, thanks! 🙏🐝

File: USERS.md [hunk 1]
```diff
@@ -385,6 +391,12 @@ Users (Alphabetically)
       L: https://www.concur.com, https://gardener.cloud/, https://github.com/gardener/gardener-extension-networking-cilium
       Q: @dragan (SAP Concur), @docktofuture & @ScheererJ (Gardener)
 
+    * N: Sapian
+      D: Sapian uses Cilium as default CNI in our product DialBox Cloud, DialBox cloud is a Edge Kubernetes cluster using [kilo](https://github.com/squat/kilo) for wirguard mesh connectivity inter-nodes. Cilium is crucial for low latancy on realtime communincations enviroments.
+      U: CNI, Network Policies, Hubble, kube-proxy replacement
+      L: https://sapian.com.co, https://arapgon.co/blog/k8s-edge
```
Comment (Preisschild) [inline USERS.md]: It appears that you have a typo in your personal websites url

**Comment by xmulligan** [comment]
@arpagon would you mind squashing this down into one commit?

**Pull request closed without merging**